src/cli.rs

```diff
@@ -0,0 +1,211 @@
+use esp_idf_svc::nvs::{EspDefaultNvsPartition, EspNvs, NvsDefault};
+
+use crate::{
+    packet::{Action, Channel, Packet},
+    queue::Queue,
+};
+
+/// Print help message
+fn print_help() {
+    println!(
+        r#"Available commands:
+  help              : Print this help page
+  id 0-65535        : Set the id of this transmitter
+  channel 0-2       : Set the channel of this transmitter
+  intensity 0-99    : Set the intensity of the command
+  shock [0-99]      : Set the command type to zapping with the given intensity
+  vibrate [0-99]    : Set the command type to good vibrations with the given
+                      intensity
+  beep              : Set the command type to make beepy noises
+  light             : Transmit a light toggle command
+  transmit [1-1000] : Transmit the configured command the given amount (default 4)
+  "#
+    );
+}
+
+/// State of the interactive shell
+pub struct State {
+    /// Shocker ID
+    pub id: u16,
+    /// Channel to transmit on
+    pub channel: Channel,
+    /// Intensity of the command
+    pub intensity: u8,
+    /// Action to perform
+    pub action: Action,
+
+    /// Last light
+    pub last_light_state: bool,
```

Comment:
I didn't think about controlling multiple devices.. I see the issue, I will revert the code to what it was previously. Won't clamp the intensities though, because that just seems like it would be confusing to the end user.